### ISSUE TITLE
Flagging invalid static method call

### ISSUE THREAD

**Issue by objeck** [objeck/objeck-lang#20]
The following should generate an error

```
class Program {
	function : Main(args: String[]) ~ Nil {
	}

	method : Bar() ~ Nil {
		Foo->Next();
	}
}

class Foo {
	New() {
	}

	method : public : Next() ~ Foo {
		return Nil;
	}
}
```

**Comment by objeck** [comment]
Fixed in 8050756

**Comment by objeck** [comment]
fixed in v5.7.2
